App.js

```diff
@@ -1,26 +1,326 @@
+import styled from 'styled-components/native'
+import { View, Vibration } from 'react-native'
+import LottieView from 'lottie-react-native'
+
 
 const App = () => {
+  const [question, setQuestion] = useState('begin')
+  const [score, setScore] = useState(0)
+  const [hide, setHide] = useState(false)
+  const [right, setRight] = useState(false)
+  const [wrong, setWrong] = useState(false)
+  const duration = 600
+
+  const afterAnswering = () => {
+    setHide(false)
+    setRight(false)
+    setWrong(false)
+  }
+
+  const nextQuestion = () => {
+    if(question==='begin') {
+      setQuestion('one')
+    } else if(question==='one') {
+      setQuestion('two')
+      afterAnswering()
+    } else if(question==='two') {
+      setQuestion('three')
+      afterAnswering()
+    } else if(question==='three') {
+      setQuestion('four')
+      afterAnswering()
+    } else if(question==='four') {
+      setQuestion('five')
+      afterAnswering()
+    } else if(question==='five') {
+      setQuestion('summary')
+      afterAnswering()
+    }
+  }
+
+  const rightAnswer = () => {
+    setScore(score+1)
+    setHide(true)
+    setRight(true)
+  }
+
+  const wrongAnswer = () => {
+    setHide(true)
+    setWrong(true)
+    return Vibration.vibrate(duration)
+  }
+
+  const restartQuiz = () => {
+    setScore(0)
+    setQuestion('begin')
+  }
+
+
   return (
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+      {question === 'begin' && (
+      <View>
+        <Container>
+        <Title>Welcome to Michels movie quiz</Title>
+        <Title>🍿🍿🍿</Title>
+        <Title>❓❓❓</Title>
+        <Button>
+          <ButtonText onPress={() => setQuestion('one')}>Start the quiz</ButtonText>
+        </Button>
+        <LottieView source={require('./1879-movie-loading.json')} autoPlay loop style={{height: 100}}/>
+        </Container>
+      </View>
+    )}
+     
+    {question === 'one' && (
+      <View>
+        <Container>
+        <Title>Question 1</Title>
+        <Title>How many movies with Leonardo DiCaprio have Martin Scorsese directed?</Title>
+        {!hide && (
+        <View>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Three</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={rightAnswer}>Five</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Four</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Two</ButtonText>
+          </Button>
+        </View>
+        )}
+        {right && (
+          <TitleRight>Correct answer! Please go to the next question</TitleRight>
+        )}
+        {wrong && (
+          <TitleWrong>Wrong answer! Please go to the next question</TitleWrong>
+        )}
+        <Score>{score} points</Score>
+        </Container>
+      </View>
```

Comment:
You could make this into a question component and import it with the props:
title, question, rightAnswerIndex or similar and then import a <Question /> here instead of duplicating the code.